src/hpc/tls.h

```diff
@@ -1,3 +1,27 @@
+/////////////////////////////////////////////////////////////////////////////
+//
+//  This file is part of GAP, a system for computational discrete algebra
+//
+//  SPDX-License-Identifier: GPL-2.0-or-later
+//
+//  Copyright of GAP belongs to its developers, whose names are too numerous
+//  to list here. Please refer to the COPYRIGHT file for details.
+//
+//  GAP is free software; you can redistribute it and/or modify it under
+//  the terms of the GNU General Public License as published by the Free
+//  Software Foundation; either version 2 of the License, or (at your
+//  option) any later version.
```

Comment:
For C/C++ files, we could also consider using inline comments `//` instead of multi-line comments. But really, it seems simpler (and more consistent) to keep the current formatting.